app/assets/javascripts/discourse/app/static/wizard/components/fields/components/index.js

```diff
@@ -0,0 +1,5 @@
+import StylingPreview from "./styling-preview";
+
+export default {
+  "styling-preview": StylingPreview,
+};
```

Comment:
Since this is the only thing left, we may want to consider getting rid of the `type: "component"` infrastructure and just do `type: "styling-preview"`?